decidim-regulations/app/controllers/decidim/regulations/regulation_controller.rb

```diff
@@ -13,7 +13,7 @@ class RegulationController < Decidim::ParticipatoryProcesses::ApplicationControl
       helper Decidim::WidgetUrlsHelper
       helper Decidim::SanitizeHelper
 
-      helper_method :collection, :promoted_participatory_processes, :participatory_processes, :stats, :filter, :types, :departments, :categories, :has_debats, :is_subcategory
+      helper_method :collection, :promoted_participatory_processes, :participatory_processes, :stats, :filter, :types, :departments
```

Comment:
is `:has_debats` related with categories?

Reply:
it was and old method, unused in this version